contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1823,6 +1823,15 @@ exec_stmt_exec_sp(PLtsql_execstate *estate, PLtsql_stmt_exec_sp *stmt)
 	char	   *querystr;
 	int			ret = 0;
 
+	/* T-SQL doesn't allow procedure calls in a function */
+	if (estate->func && estate->func->fn_oid != InvalidOid && estate->func->fn_prokind == PROKIND_FUNCTION && estate->func->fn_is_trigger == PLTSQL_NOT_TRIGGER)
+	/* check EXEC is running in the body of function */
```

Comment:
This comment is redundant in this case since we arent using is_scalar_func

Reply:
removed